cron/session.php

```diff
@@ -0,0 +1,79 @@
+#!@PHP_BIN@
+<?php
+
+/*
+ * Copyright 2005 - 2021 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+require_once realpath(__DIR__ . "/../config/centreon.config.php");
+include_once _CENTREON_PATH_ . "/cron/centAcl-Func.php";
+include_once _CENTREON_PATH_ . "/www/class/centreonDB.class.php";
+include_once _CENTREON_PATH_ . "/www/class/centreonLog.class.php";
+
+$pearDB = new CentreonDB();
+$centreonLog = new CentreonUserLog(0, $pearDB);
+
+// Default session duration
+define('SESSION_DEFAULT_DURATION', 120);
+
+try {
+    /**
+     * Remove expired sessions
+     */
+    // Find duration of session_expiration
+    $sessionDuration = SESSION_DEFAULT_DURATION;
+    $durationQuery = $pearDB->query("SELECT `value` from options where `key` = 'session_expire'");
+    if (($duration = $durationQuery->fetch(\PDO::FETCH_ASSOC)) !== false) {
+        $sessionDuration = $duration['value'];
+    } else {
+        // cannot find the default value. setting a new value
+        $pearDB->query("DELETE FROM options WHERE `key` = 'session_expire'");
```

Comment:
why trying to delete a value if you have already checked it does not exist ?

Reply:
I checked that the value doesn't exist.
But if the key does not too, I won't be able to update the value. And if the key exists I won't be able to insert a new value

I prefer to clean the data before setting a new default value.
But this case should not occurs unless the options table is corrupted